front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PipelineController.groovy

```diff
@@ -117,6 +117,19 @@ class PipelineController {
     pipelineDAO.bulkImport(pipelines)
   }
 
+  @PreAuthorize("@fiatPermissionEvaluator.storeWholePermission() and hasPermission(#application, 'APPLICATION', 'WRITE')")
```

Comment:
@ajordens Wondering what you think the best permission check would be for re-ordering?

Reply:
The tricky thing is that it really needs to be checked on each of the pipeline ids and not the application that was supplied.

Might require explicitly using the `FiatPermissionEvaluator` after each pipeline is loaded to verify that the user has WRITE on the associated application.

(or you need to check that each pipeline is actually associated with the supplied application)